aws/ami/build_deb_ami.sh

```diff
@@ -123,8 +123,11 @@ SSH_USERNAME=ubuntu
 
```

Comment:
Commit subject line as an extra "a" at the beginning.

Reply:
removed